setup.py

```diff
@@ -1,17 +1,25 @@
 from setuptools import setup
 
+try:
+    import pypandoc
+    long_description = pypandoc.convert('README.md', 'rst')
+except(IOError, ImportError):
+    long_description = open('README.md').read()
+
```

Comment:
Just noting that you can use `setup.cfg` as noted here to point to `README.md` and PyPI will handle it from there, apparently. http://peterdowns.com/posts/first-time-with-pypi.html:

> Prepare your package
>
> Every package on PyPI needs to have a file called `setup.py` at the root of the directory. If your'e using a markdown-formatted read me file you'll also need a `setup.cfg` file.

and further down:
> setup.cfg
>
> This tells PyPI where your README file is.
>
> `[metadata]`
> `description-file = README.md`
>
> This is necessary if you're using a markdown readme file. At upload time, you may still get some errors about the lack of a readme — don't worry about it. If you don't have to use a markdown README file, I would recommend using reStructuredText (REST) instead.